__tests__/app.test.js

```diff
@@ -35,50 +35,56 @@ describe('app routes', () => {
     
           name: 'turkey neck',
           cool_factor: 3,
-          cool_haircut: false
-          
+          cool_haircut: false,
+          id: 1
```

Comment:
You don't need to add ids in to your data since sql will do it for you in your create tables file. That allows you to add more data later (like in lab 7) and sql will handle adding an id to it automatically